runtime/api/WeexInstance.js

```diff
@@ -53,9 +53,10 @@ function moduleSetter (id, module, method, fn) {
 }
 
 export default class WeexInstance {
-  constructor (id, config) {
+  constructor (id, config, data) {
     setId(this, String(id))
     this.config = config || {}
+    this.data = data || {}
```

Comment:
Using `data` as the property is too common and ambiguous. Maybe you should use a more explicit and semantic name here.